### PR TITLE
[ADF-4856] Core refactor - be able to run ng build

### PR DESCRIPTION
**Pull request by mauriziovitale**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/src/lib/viewer/viewer.module.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ import { A11yModule } from '@angular/cdk/a11y';
         ExtensionsModule,
         CommonModule,
         MaterialModule,
-        TranslateModule.forChild(),
```
Comment (DenysVuika) [inline lib/core/src/lib/viewer/viewer.module.ts]: this should be a problem for lazy loading

Comment (mauriziovitale) [reply lib/core/src/lib/viewer/viewer.module.ts]: Put it back

File: lib/core/src/lib/settings/host-settings.module.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ import { HostSettingsComponent } from './host-settings.component';
     imports: [
         CommonModule,
         MaterialModule,
-        TranslateModule.forChild(),
+        TranslateModule,
```
Comment (DenysVuika) [inline lib/core/src/lib/settings/host-settings.module.ts]: Needs to be .forChild

**Comment by eromano** [comment]
it needs to be redone

**Pull request closed without merging**